### PR TITLE
Sign-in and Login functions

### PR DESCRIPTION
**Pull request by nozomi-matsuoka**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+			department: params[:user][:department],
+			admin: params[:user][:admin]
+			)
+
+		if @user.save
+			redirect_to root_url
+		else
+			render "new"
+		end
+	end
```
Comment (inahsanjuan) [inline app/controllers/users_controller.rb:15]: @nozomi-matsuoka
For this project I want us to use **strong parameters** . I will discuss more on this to the class.
https://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters/jp 


In this file, please provide a private method at the end of the class: 
```
private
  def user_params
    params.require(:user).permit(:name, :email, :password, :password_confirmation, :department, :admin)
  end
```

Then change `line 8-15` to:
`@user = User.new(user_params`)

File: app/helpers/sessions_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module SessionsHelper
```
Comment (inahsanjuan) [inline app/helpers/sessions_helper.rb:1]: @nozomi-matsuoka 
Let's try to create methods for log in.
Create a method here for sessions:
```
def log_in(user)
  session[:user_id] = user.id
end
```

Then you can use this method in SessionsController : 
https://github.com/framgia831/rails-elearning-nozomi/pull/1/files#diff-d5241d488259f32ecbe2f636133e5ddaR6

File: app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+
+	def new
+	end
+
+	def create
+		user = 	User.find_by(email: params[:session][:email])
+
+		if  user && user.authenticate(params[:session][:password])
+			session[:user_id] = user.id
```
Comment (inahsanjuan) [inline app/controllers/sessions_controller.rb]: @nozomi-matsuoka Change this line to : 
`log_in user`
If you finished to add method in **SessionsHelper**

**Review by inahsanjuan — CHANGES_REQUESTED** [review]

File: app/controllers/sessions_controller.rb (hunk 3 above)
Comment (nozomi-matsuoka) [reply app/controllers/sessions_controller.rb]: I have a question.
Do you mean that the line 10(session[:user_id] = user.id) should replace with [log_in user] ?
I tried it, but there is an error.

**Pull request merged**